probe.go

```diff
@@ -801,6 +801,140 @@ func probeLinkMonitor(c FortiHTTP) ([]prometheus.Metric, bool) {
 	return m, true
 }
 
+func probeVirtualWanPerf(c FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		mLink = prometheus.NewDesc(
+			"fortigate_virtualwan_healthcheck_status",
+			"Whether the link is up or not",
+			[]string{"vdom","sla", "interface", "state"}, nil,
+		)
+		mLatency = prometheus.NewDesc(
+			"fortigate_virtualwan_healthcheck_latency",
+			"Measured latency",
```

Comment:
A little more details would be nice here for the users.  As this is basically all the user knows what this values mean.     
See the LinkMonitor for what we have found through experimentation what these values represent.    

Same goes for `jitter` and `packet loss`